src/components/TodoComponents/TodoData.js

```diff
@@ -0,0 +1,24 @@
+//toDo array
+
+export const todoArr = [
```

Comment:
I like that you turned your data into a component. That makes your code easier to modify and understand.